index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" media="screen" href="./styles.css">
+    </head>
+
+    <body>
+        <!--<section id="portfolio" class="project-section">-->
```

Comment:
- [ ] As mentioned by the previous reviewer, the previous project milestone - the toolbar and headline section is missing in your project. This portfolio's "project section" and "about me" sections are not stand-alone projects. Please add the navbar/toolbar and the headline section. Please view [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=KHwEU5HcxaGR5S6T-0)